src/test/java/com/urbanairship/api/client/APIClientTest.java

```diff
@@ -1881,6 +1881,130 @@ public void testListAllChannels() {
+                "      \"device_type\": \"ios\",\n" +
+                "      \"installed\": true,\n" +
+                "      \"opt_in\": true,\n" +
+                "      \"background\": true,\n" +
+                "      \"push_address\": \"E4EA0D96092A9213BB186BEF66E83EE226401F82B3A77A1AC8217A8FE8ED4614\",\n" +
+                "      \"created\": \"2014-07-09T18:08:37\",\n" +
+                "      \"last_registration\": \"2014-10-02T01:41:42\",\n" +
+                "      \"alias\": null,\n" +
+                "      \"tags\": [\n" +
+                "        \"version_1.5.0\"\n" +
+                "      ],\n" +
+                "      \"ios\": {\n" +
+                "        \"badge\": 1,\n" +
+                "        \"quiettime\": {\n" +
+                "          \"start\": \"17:00\",\n" +
+                "          \"end\": \"9:00\"\n" +
+                "        },\n" +
+                "        \"tz\": \"America\\/Los_Angeles\"\n" +
+                "      }\n" +
+                "    },\n" +
+                "    {\n" +
+                "      \"channel_id\": \"00d8cb94-eac9-49fb-bad0-29298a06730e\",\n" +
+                "      \"device_type\": \"ios\",\n" +
+                "      \"installed\": false,\n" +
+                "      \"opt_in\": false,\n" +
+                "      \"push_address\": \"21F34C9ED37EAF8D7DC43561C07AA398CA5C6F503196C9E8230C50C0959B8653\",\n" +
+                "      \"created\": \"2014-02-22T22:48:37\",\n" +
+                "      \"last_registration\": null,\n" +
+                "      \"alias\": \"iPhone 7,1\",\n" +
+                "      \"tags\": [\n" +
+                "        \"kablam\",\n" +
+                "        \"version_1.3\"\n" +
+                "      ],\n" +
+                "      \"ios\": {\n" +
+                "        \"badge\": 1,\n" +
+                "        \"quiettime\": {\n" +
+                "          \"start\": null,\n" +
+                "          \"end\": null\n" +
+                "        },\n" +
+                "        \"tz\": null\n" +
+                "      }\n" +
+                "    },\n" +
+                "    {\n" +
+                "      \"channel_id\": \"01257ecd-8182-41fe-a741-9fed91b993cb\",\n" +
+                "      \"device_type\": \"android\",\n" +
+                "      \"installed\": false,\n" +
+                "      \"opt_in\": false,\n" +
+                "      \"push_address\": null,\n" +
+                "      \"created\": \"2013-01-25T00:55:05\",\n" +
+                "      \"last_registration\": null,\n" +
+                "      \"alias\": null,\n" +
+                "      \"tags\": [\n" +
+                "        \n" +
+                "      ]\n" +
+                "    }\n" +
+                "  ],\n" +
+                "  \"next_page\": \"https:\\/\\/go.urbanairship.com\\/api\\/channels?limit=5&start=0143e4d6-724c-4fc8-bbc6-ca647b8993bf\"\n" +
```

Comment:
For the sake of this test also being an implementation example, this `next_page` url would not be returned by itself as it's passed into `client.listAllChannels` below.  You might want to generate a different random uuid to get returned here in the url.